lib/core/include/irods/irods_configuration_keywords.hpp

```diff
@@ -164,6 +164,7 @@ namespace irods
     extern const char* const KW_CFG_SERVICE_ROLE_PROXY;
 
     extern const char* const KW_CFG_IRODS_PLUGIN_DIRECTORY;
+    extern const char* const KW_CFG_PLUGIN_DIRECTORY;
```

Comment:
I feel like an explanatory comment here would go a long way. I had to go to the issue in the commit message to understand what the difference was between these two and even now I'm not sure that I do. Is `KW_CFG_IRODS_PLUGIN_DIRECTORY` used in server_config or irods_environment.json?

Reply:
I've been thinking maybe we should remove plugin_directory entirely. I can't see why someone would have a separate universe of shared libraries sitting on the side of the ones shipped with the server.

At that point, they can just compile the software from source.

Reply:
Leaving in. Can remove in iRODS 6 if needed.

Will add a comment.

Reply:
Added comment.